### DISCUSSION TITLE
Laravel 10.x vite compatibility issue.

### DISCUSSION THREAD

**Discussion by smartdatasoft** [opcodesio/log-viewer#246] [Ideas]
when we try to access the route file it shows this issue. in laravel 10

```app.js failed to load. Please refresh the page, re-publish Log Viewer assets, or fix routing for vendor assets.```

**Comment by arukompas** [comment]
Hey @smartdatasoft , what do you mean by "access the route file"?

**Comment by smartdatasoft** [comment]
In laravel 10 which uses vite , when we try to access the url of the log viewer, this error show in pop  and log viewer not working.

**Comment by arukompas** [comment]
@smartdatasoft can you share a screenshot of the error? Just so I understand where it's coming from, and see the stack trace.

Do other Laravel packages like Horizon work as expected?
